Update README.md (#133)
NRL-Plasma-Physics-Division/turbopy: README.md

-   Activate: `conda activate turbopy`
-   Install turboPy into the environment (from the main folder where setup.py is): 

> 	- Install turboPy in editable mode (i.e. setuptools "develop mode") if you are modifying turboPy itself: `pip install -e .` 

Reviewer: ```suggestion
-   Install turboPy into the environment in editable mode (from the main folder where setup.py is): `pip install -e .` 
```

> 	- If you just plan to develop a code using the existing turboPy framework: `pip install turbopy` 

Reviewer: If they are just using turboPy, then they should `pip install turbopy`
```suggestion
```

Reviewer: oops... made this comment above already

> 	- If you just plan to develop a code using the existing turboPy framework: `pip install .` 

Reviewer: I don't think we need this. If they want to just develop an app, then they should just use `pip install turbopy`
```suggestion
```
